src/main/java/com/github/nicholasmoser/gnt4/seq/Seq.java

```diff
@@ -611,721 +612,315 @@ public class Seq {
+    actionDescriptions.put(0x172, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x173, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x174, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x175, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x176, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x177, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x178, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x179, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x17A, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x17B, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x17C, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x17D, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x17E, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x17F, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x180, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x181, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x182, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x183, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x184, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x185, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x186, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x187, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x188, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x189, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x18A, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x18B, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x18C, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x18D, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x18E, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x18F, "(UNUSED - points to chr_act)");
+    actionDescriptions.put(0x190, "Ground Throw");
+    actionDescriptions.put(0x191, "Back Ground Throw");
+    actionDescriptions.put(0x192, "Air Throw");
+    actionDescriptions.put(0x193, "Activated X");
+    actionDescriptions.put(0x194, "Unknown");
+    actionDescriptions.put(0x195, "Activated X (Transformation)");
+    actionDescriptions.put(0x196, "Activated 2X");
+    actionDescriptions.put(0x197, "Unknown");
+
+    actionDescriptions.put(0x1D0, "Activated 3-Man X #1");
+    actionDescriptions.put(0x1D1, "Activated 3-Man X #2");
+    actionDescriptions.put(0x1D2, "Activated 3-Man X #3");
+    actionDescriptions.put(0x1D3, "Activated 3-Man X #4");
+
+    actionDescriptions.put(0x1D5, "Activated 3-Man X #5");
+
+    actionDescriptions.put(0x1DF, "Activated 3-Man X #6");
+    actionDescriptions.put(0x1E0, "Getting thrown (front)");
+    actionDescriptions.put(0x1E1, "Getting thrown (back)");
+    actionDescriptions.put(0x1E2, "Unknown");
```

Comment:
I think this one should be "Getting thrown (air)", even if the act id listed in game is 11E2 and not 1E2

Reply:
Pretty sure about this, getting hit by super is listed as 11E3 as well as getting thrown is listed as 11E0 and 11E1 respectively

Reply:
> even if the act id listed in game is 11E2 and not 1E2

I've noticed this as well. I believe that all actions are forced to 12 bits (0xFFF) and the higher bits represent some kind of flags. I feel like I've seen some SEQ code showing this, let me see if I can find it.

Reply:
Just did some testing and I can confirm that you are correct, nice!
Updated here: https://github.com/NicholasMoser/GNTool/pull/97/commits/06b9f521a07267e5054e5d9aeaccf2027e877018